scripts/docs/generate_population_table.pl

```diff
@@ -379,6 +379,10 @@ sub get_vcf_content_types {
     if ( ($info_field =~ /AF=/) || ($info_field =~ /AC=/ && $info_field =~ /AN=/) ) {
       push @types, "frequency";
     }
+    # a hard-coded check for NCBI-ALPHA and TOPMED as they have very special field for frequency
+    if ( ($info_field =~ /AN_SAMN/) || ($info_field =~ /TOPMED=/) ) {
+      push @types, "frequency";
+    }
```

Comment:
Do you think it would be better to put everything in the same `if` statement? Also, should `AN_SAMN` be `AN_SAMN=` instead?

```suggestion
    if (
      ($info_field =~ /AF=/) ||
      ($info_field =~ /AC=/ && $info_field =~ /AN=/) ||
      # hard-coded check for NCBI-ALPHA and TOPMED as they have very special field for frequency
      ($info_field =~ /AN_SAMN=/) || ($info_field =~ /TOPMED=/)
    ) {
      push @types, "frequency";
    }
```

Reply:
inline comment would not work actually.
the actual field have full sample name `AN_SAMN...=` so I did not put the equal sign here.